README.md

```diff
@@ -23,28 +23,32 @@ It has two main components:
   
 Both frontend and backend are highly flexible.
 
-As an example, *powdr* contains a frontend that enables you to write code in (no-std) Rust,
-which is compiled to RISCV, then to powdr-asm and finally to powdr-PIL.
+The [powdrVM](https://docs.powdr.org/quick_start_vm.html) is a zkVM
+implementation that builds on *powdr*.
+Users can write code in std Rust which is compiled to RISCV,
+then to powdr-asm and finally to powdr-PIL.
```

Comment:
there's both `powdr-PIL` and `powdr-pil` in the text

Reply:
not in my changes :grimacing:

Reply:
done